package.json

```diff
@@ -35,13 +35,13 @@
     "test:dev": "npm run test-unit:dev"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.10.0",
```

Comment:
Please revert all dependency updates not related to chart.js beta-9 so it will be easier to rebase the `chartjs-next` branch on top of `master` (which I still do regularly). Same for `package-lock.json`.

Reply:
Done